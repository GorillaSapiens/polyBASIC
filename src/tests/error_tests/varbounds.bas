a(22) = 1
