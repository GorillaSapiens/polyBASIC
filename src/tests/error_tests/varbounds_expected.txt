ERROR: @1:1, ARRAY INDEX OUT OF BOUNDS ❰a❱(22) ❰a❱(0..10) @1
